index.js

```diff
@@ -1,14 +1,25 @@
+require('colors');
 const program = require('commander');
 const pjson = require('./package.json');
 
 const generateComponent = require('./actions/generateComponent');
 
 program
   .version(pjson.version, '-v, --version')
+
   .command('gc <component>')
   .option('-s, --style', 'With stylesheet')
   .option('-f, --functional', 'Create functional component')
   .option('-c, --withConnect', 'Wrap with redux connect')
   .action(generateComponent);
 
+program
```

Comment:
why you use program here? it is already in line 7

Reply:
For better readability as it's another command